### PR TITLE
Add Pre-set value in a textarea within a radio

### PR DESCRIPTION
**Pull request by SriHV**
### What is the context of this PR?

#3242... Added a preset value for a radio option that has textarea

### How to review this PR
Add a preset value to one of the radio examples with a textarea option, and verify that the value is displayed.

### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

**Review by alessioventuriniAND — APPROVED** [review]

**Review by precious-onyenaucheya-ons — APPROVED** [review]

File: src/components/radios/_macro.spec.js [hunk 1]
```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
     },
 };
 
+const EXAMPLE_RADIO_ITEM_WITH_PRESET_TEXTAREA = {
```
Comment (rmccar) [inline src/components/radios/_macro.spec.js:125]:
```suggestion
const EXAMPLE_RADIO_ITEM_TEXTAREA_WITH_PRESET_VALUE = {
```
More consistent with other example names

File: src/components/radios/_macro.spec.js [hunk 2]
```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
...
+    other: {
+        otherType: 'textarea',
+        checked: true,
+        id: 'other-textbox-radio-with-text-input',
```
Comment (rmccar) [inline src/components/radios/_macro.spec.js:129]:
```suggestion
        id: 'other-textarea-input',
```
Should be named something like this to be consistent with other examples and it specifies its a textarea not a text input

File: src/components/radios/_macro.spec.js [hunk 3]
```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
...
+        name: 'other-answer',
+        value: 'other-answer',
```
Comment (rmccar) [inline src/components/radios/_macro.spec.js:131]:
```suggestion
        value: 'other answer',
```
This should be an answer someone would write in the text area so for it to be a good example it shouldn't be hyphenated

**Review by rmccar — CHANGES_REQUESTED** [review]
Some minor changes

**Review by rmccar — APPROVED** [review]

**Pull request merged**